core/src/main/resources/swagger/api-docs.yaml

```diff
@@ -1394,6 +1394,42 @@ paths:
         500:
           $ref: '#/components/responses/RawlsInternalError'
       x-codegen-request-body-name: newEntityNameJson
+  /api/workspaces/{workspaceNamespace}/{workspaceName}/entityType/rename:
```

Comment:
Up for debate: what about making this 
`POST /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/renameType`?

It feels like this API should be parallel to 
`POST /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/delete`

alternately, it could be
`POST /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/{entityType}/rename`, where the oldName is specified in the path instead of the post body, but I like that less.

Reply:
I went with `POST /api/workspaces/{workspaceNamespace}/{workspaceName}/entities/renameEntityType`